frontend/src/pages/modelServing/screens/projects/ServingRuntimeList.tsx

```diff
@@ -63,12 +63,7 @@ const ServingRuntimeList: React.FC = () => {
           />
         }
       >
-        <ServingRuntimeTable
```

Comment:
Ok, I can agree with this, kinda weird that part of the rendered data is fetched with  context and the other part with props, I can see we either do one thing or another.

Reply:
Yes, what I meant with that is that we are supposed to have "pure" components without context data being rendered, I guess that in this case, since this component is targeted for an specific page with no intention of using it outside projects it should be fine to just use contexts.